packages/reaction-core/client/helpers/apps.js

```diff
@@ -165,8 +165,19 @@ function getReactionApps(optionHash) {
                 if (!registry.packageName) registry.packageName = app.name;
                 if (registry.enabled !== false) {
                   // TODO move this to a function and reuse with Template.dashboardHeader.helpers
-                  const registryLabel = registry.label ? registry.label.toCamelCase() : "";
-                  const i18nKey = `admin.${registry.provides}.${registryLabel}`;
+                  let registryLabel = "";
+                  let i18nKey;
+                  // first we check the default place for a label
+                  if (registry.label) {
+                    registryLabel = registry.label.toCamelCase();
+                    i18nKey = `admin.${registry.provides}.${registryLabel}`;
+                    // and if we don't find it, we are trying to look at first
```

Comment:
Could you provide cases, when registry.label undefined? Why do want to take just first registry entry? It could lead to confusing for developer or translator what i18nKey is generated for the entry. I propose to keep them as simple as possible